### ISSUE TITLE
Installation with Homebrew

### ISSUE THREAD

**Issue by MaxDesiatov** [WeTransfer/GitBuddy#36]
Hello, and thanks for this great tool!

It would be great if GitBuddy could be installed via Homebrew, which would be one of the easiest ways to install it on macOS. I know it can be installed with Mint, but for machines that don't have Mint installed, this is a two step process, which is less than ideal. There is also a use case for installing GitBuddy on CI as a part of build process triggered on a new Git tag, where one would expect the build process to finish as quickly as possible, which is much easier to achieve with Homebrew than with Mint.

**Comment by AvdLee** [comment]
I totally agree with you, this would be great to have! However, adding support for Swift packages isn't super nice as it requires to create certain "taps" like explained here: https://nshipster.com/homebrew/

I'm happy to open such a tap repository but it's not a priority for us right now to work on this.
